lib/webauth.js

```diff
@@ -172,10 +200,15 @@ module.exports = function(authManager) {
         }
         result.addHandlerResult(handlerResult, handler);
       }
-      result.updateStatus();
+      
+      result.updateStatus(authManager.sessionTimeoutMs);
+      if (result.expms !== -1) {
```

Comment:
is this `-1` some special magic value? where is it supposed to come from?

Reply:
I will add a comment: -1 means they explicitly want there to be no expiration.

Reply:
Maybe we need to also define a constant for it...